libraries/typescript/packages/evals/src/shared/errors.ts

```
@@ -0,0 +1,36 @@
+export class EvalError extends Error {
```

Biome (2.1.2)

[error] Do not shadow the global "EvalError" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)